hw/drivers/pwm/pwm_nrf52/src/pwm_nrf52.c

```diff
@@ -437,7 +440,7 @@ nrf52_pwm_configure_channel(struct pwm_dev *dev,
     struct nrf52_pwm_dev_global *instance = &instances[inst_id];
     nrfx_pwm_config_t *config = &instance->config;
 
-    if (!instance->in_use) {
+    if (!instance->) {
```

Comment:
funny syntax!

Reply:
heh

Reply:
@mlaz were you trying to alter the check here, or did the in_use just get lost.

Reply:
It did get lost, apparently before I commit, and after I compiled.

Reply:
https://github.com/jacobrosenthal/mynewt-core/pull/2 @jacobrosenthal